### ISSUE TITLE
Blank screen when Content-Type is wrong

### ISSUE THREAD

**Issue by csteinle92** [usebruno/bruno#513]
Steps to reproduce:
- Create a simple GET REST endpoint which returns a JSON data structure
- Leave out the Content-Type "application/json" for the response (in PHP that happened by default and it's "text/html; charset=UTF-8")
- Call that endpoint via the bruno UI client

Expected result:
- Some error or anything indicating that something went wrong

Actual result:
- The screen of the UI blanks completely and doesn't allow any interaction anymore. Only a restart of the UI helps in that case.
- In the DevTools console the following error can be seen:
```
TypeError: r.replace is not a function
    at index-cf839caf38d3b951.js:1:85181
    at Object.useMemo (framework-49f1e091cbf6b261.js:1:69562)
    at n.useMemo (framework-49f1e091cbf6b261.js:1:136649)
    at Yt (index-cf839caf38d3b951.js:1:85088)
    at Eo (framework-49f1e091cbf6b261.js:1:63194)
    at xi (framework-49f1e091cbf6b261.js:1:120434)
    at bs (framework-49f1e091cbf6b261.js:1:109370)
    at gs (framework-49f1e091cbf6b261.js:1:109298)
    at vs (framework-49f1e091cbf6b261.js:1:109161)
    at as (framework-49f1e091cbf6b261.js:1:105941)
```

The location in the minified JS code found through the DevTools:
![image](https://github.com/usebruno/bruno/assets/17760559/bac0623a-2c97-4509-875b-5212638e8c42)

**Comment by Its-treason** [comment]
This bug should be fixed in Version 0.22.1, are you on the newest version?

**Comment by csteinle92** [comment]
Oh, I was on 0.21.1. Now I updated to 0.22.1 and it works.
Also just found #382 in the release notes. Thanks for pointing out!

**Comment by csteinle92** [comment]
Closed as it is a duplicate of #382.
